deluge/core/torrent.py

```diff
@@ -838,9 +841,14 @@ def get_peers(self):
                 client = 'unknown'
 
             try:
-                country = component.get('Core').geoip_instance.country_code_by_addr(
-                    peer.ip[0]
-                )
+                #if it's an I2P peer use I2P logo as flag
+                #added ip.png to ui/data/pixmaps/flags
```

Comment:
Should be removed since should not be describing changes in comments, those are for commit messages